<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>repair</class>
 <widget class="QDialog" name="repair">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDateTimeEdit" name="dateTimeEdit_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>280</y>
     <width>194</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>431</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Начать ремонт</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>250</y>
     <width>194</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>250</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Начало ремонта</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>280</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Конец ремонта</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>320</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Тип ремонта</string>
   </property>
  </widget>
  <widget class="QTableView" name="machineview">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>441</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="repairview">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>321</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>320</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
